content/en/agent/proxy.md

```diff
@@ -418,6 +451,12 @@ database_monitoring:
 network_devices:
     metadata:
         dd_url: haproxy.example.com:3841
+    snmp_traps:
+        forwarder:
+            dd_url: haproxy.example.com:3842
+    netflow:
```

Comment:
👋 @vickenty thanks for this PR. Looks good for `snmp_traps`, but for `netflow` it's a bit too early, the feature is not GA yet. I think it's preferred to add `netflow` once it's GA.

Reply:
If ok for you, I can create a Jira card on our board to handle that when netflow is GA.

Reply:
cc @gmarabout

Reply:
Fixed! If you make a card please cc me!